Updated for Swift 3

anfriis opened the pull request:
And created a new release

anfriis committed:
Update to Swift 3

anfriis committed:
Update Storyboard to Xcode 8

anfriis committed:
Update Pod version

anfriis committed:
Update README with new version number

hsusmita commented:
@Anders123fr Thanks for this PR @Anders123fr. Can you also update the CHANGELOG.md?
Thank you!

hsusmita commented on GrowingTextViewHandler/GrowingTextViewHandler/Source/GrowingTextViewHandler.swift:
```diff
@@ -8,60 +8,60 @@
 	}
 
-	private var textViewHeightConstraint: NSLayoutConstraint?
+	fileprivate var textViewHeightConstraint: NSLayoutConstraint?
```
@Anders123fr What is the rationale behind using `filePrivate` instead of `private`? I believe, these variables should be accessible from this declaration only.

anfriis replied:
I just ran the Migrator, but I can change it to private if you think

hsusmita replied:
I think private would be fine. Please change that.

hsusmita commented on GrowingTextViewHandler/GrowingTextViewHandler/Source/GrowingTextViewHandler.swift:
```diff
@@ -8,60 +8,60 @@
+	open var heightChangeAnimationDuration = 0.5
+	open var maximumNumberOfLines = Int.max {
 		didSet {
 			updateInitialHeightAndResize()
```
:+1:

anfriis committed:
Make build schema shared to support Carthage

anfriis committed:
Move project files to root

anfriis committed:
Carthage support

hsusmita commented:
@Anders123fr Can you rebase your branch with master?

anfriis committed:
Set Deployment target to 8

hsusmita commented:
@Anders123fr After merging your branch locally, I am getting the following warning. Can you look into this issue?
<img width="371" alt="screen shot 2016-11-09 at 6 13 46 pm" src="https://cloud.githubusercontent.com/assets/3590619/20138806/86dd6d1a-a6a8-11e6-9bef-2779b6c93dc1.png">

anfriis committed:
Fix warnings

anfriis commented:
I changed to private and fixed the warnings, except for the Single Line Etched warning... deleted every setting related to it but couldn't get rid of it. 
Is that enough to merge?
